sdk/communication/azure-communication-callingserver/swagger/README.md

```diff
@@ -32,8 +32,9 @@ To update generated files for calling service, run the following command
 ### Code generation settings
 ``` yaml
 tag: package-2022-04-07-preview
+use: '@autorest/java@4.1.33'
 require:
-    - https://github.com/richardcho-msft/azure-rest-api-specs/blob/dev-communication-CallingServer-2022-04-07-preview/specification/communication/data-plane/CallingServer/readme.md
+    - https://github.com/Azure/azure-rest-api-specs/blob/anuchan/add-callingserver-2022-04-07-preview/specification/communication/data-plane/CallingServer/readme.md
```

Comment:
The swagger link was pointing to someone's personal repo that no longer exists. But I found a copy in the .NET repo. We’ll be asking service team to have it in official rest-api-repo main branch, for the purpose of this PR, I pushed it to a non-main branch in official rest-api-repo